xds/src/main/java/io/grpc/xds/EnvoyProtoData.java

```diff
@@ -299,18 +322,31 @@ public String toString() {
     }
 
     static LbEndpoint fromEnvoyProtoLbEndpoint(
+        io.envoyproxy.envoy.config.endpoint.v3.LbEndpoint proto) {
+      io.envoyproxy.envoy.config.core.v3.SocketAddress socketAddress =
+          proto.getEndpoint().getAddress().getSocketAddress();
+      InetSocketAddress addr =
+          new InetSocketAddress(socketAddress.getAddress(), socketAddress.getPortValue());
+      return new LbEndpoint(
+          new EquivalentAddressGroup(ImmutableList.<java.net.SocketAddress>of(addr)),
+          proto.getLoadBalancingWeight().getValue(),
+          proto.getHealthStatus() == io.envoyproxy.envoy.config.core.v3.HealthStatus.HEALTHY
+              || proto.getHealthStatus()
+                  == io.envoyproxy.envoy.config.core.v3.HealthStatus.UNKNOWN);
```

Comment:
nit: this line can be shrunk to the previous one. Same for below.

Reply:
Done for below. Though this line can not be shrunk because otherwise the length would be 101 chars.